src/include/btmem.h

```diff
@@ -17,11 +17,12 @@
 #define WT_READ_NO_WAIT 0x0020u
 #define WT_READ_PREV 0x0040u
 #define WT_READ_RESTART_OK 0x0080u
-#define WT_READ_SKIP_DELETED 0x0100u
-#define WT_READ_SKIP_INTL 0x0200u
-#define WT_READ_TRUNCATE 0x0400u
-#define WT_READ_VISIBLE_ALL 0x0800u
-#define WT_READ_WONT_NEED 0x1000u
+#define WT_READ_SEE_DELETED 0x0100u
```

Comment:
Style question: We have a `WT_CURSTD_IGNORE_TOMBSTONE` flag in WiredTiger in our public API. Should that be `CURSTD_IGNORE_DELETED` or should these flags be `SEE_TOMBSTONE` and `SKIP_TOMBSTONE`? I think we should aim to be consistent one way or another. 

As we're already in the situation where we have both in the codebase I'm okay with this PR regardless but think its worth discussing.

Reply:
I agree with the sentiment and expectation of where to fix it. FWIW: I generally (strongly) prefer positively named flags (i.e: see instead of ignore in this case). I find with negatives I end up getting myself confused about their meaning when they get negated.

Reply:
I prefer positive names too. Those `CURSTD` flags are in the public-facing API - what's the process (if any) for making this sort of breaking change? (I'm not sure how much sense it makes to expose tombstones to users, I'll need some guidance here.)

Reply:
There isn't a process, I think we've been changing them whenever we want 😓 . I don't think we should expose tombstones to the user so i'd probably make it `DELETED`. Would the flag then be `CURSTD_READ_DELETED_CONTENT`. Leaving out `CONTENT` makes it seem a bit less obvious as to what is being read.

Reply:
Made WT-11683 for the flags tidy-up.